src/prelude.rs

```diff
@@ -57,6 +57,7 @@ pub use app_chooser::AppChooserExt;
 pub use color_chooser::ColorChooserExt;
 pub use dialog::DialogExt;
 pub use tree_model::TreeModelExt;
+pub use tree_sortable::{SortColumn, TreeSortableExt};
```

Comment:
We don't need `SortColumn` in the prelude, it's not important enough. But it should be reexported in `lib.rs`.